### DISCUSSION TITLE
CKAN version: 2.10: Upload to dataStore pending even when the logged content shows Express Load completed after upgrading to CKAN 2.10.0

### DISCUSSION THREAD

**Discussion by namwanza** [ckan/ckan#7418] [Q&A]
# Situation 1
The express and manual express loading of resources (CSV or xlx) into dataStore shows pending status and on checking the logged details, xloader shows express loading was completed. 

# Situaltion 2
No available Views for any Resource. My plugins are:

`ckan.plugins = resource_proxy viewhelpers customised_theme scheming_datasets datatables_view harvest ckan_harvester dcat dcat_rdf_harvester dcat_json_harvester dcat_json_interface structured_data stats text_view image_view recline_view recline_map_view recline_graph_view recline_grid_view video_view audio_view pdf_view wmts_view shp_view navigablemap choroplethmap linechart barchart piechart dashboard_preview officedocs_view multilingual_tag multilingual_group example_organization_label pages showcase dataexplorer_view dataexplorer_table_view dataexplorer_chart_view dataexplorer_map_view dataexplorer_web_view geo_view geojson_view datastore xloader`

![country-s-employee-records-Text-sample-xlx-files-ATB-Open-Data](https://user-images.githubusercontent.com/32957478/221131607-dfa073fd-8749-461a-9d80-67a820102d97.png)

**Comment by lgmatabh** [comment]
Hi all,

same situation (Situation 1 above) with CKAN 2.9.8 and xloader 0.12.2.

CKAN 2.9.8 with xloader 0.8.0 the status stay "complete".

https://ckan-ds.pbh.gov.br/api/3/action/status_show ->

{"help": "https://ckan-ds.pbh.gov.br/api/3/action/help_show?name=status_show", "success": true, "result": {"site_title": "Portal de Dados Abertos de Desenvolvimento - 2.9.8", "site_description": "", "site_url": "https://ckan-ds.pbh.gov.br", **"ckan_version": "2.9.8",** "error_emails_to": null, "locale_default": "pt_BR", "extensions": ["xloader", "datatables_view", "datastore", "text_view", "image_view", "recline_view"]}

(default) root@vlpd-ckan03:~/scripts# pip list|grep ckanext-xloader
**ckanext-xloader    0.12.2**  



Obs: There was a code change in version 0.12.x regarding the timestamp with UTC.

![Captura de tela em 2023-02-28 16-23-28](https://user-images.githubusercontent.com/39322388/223534109-d78267b5-ae83-40e0-8871-53c55f230b11.png)
![Captura de tela em 2023-02-28 16-22-59](https://user-images.githubusercontent.com/39322388/223534113-0a44dac6-85cf-4b65-9ab6-4b88e5de5eb8.png)
